app/components/common/export.vue

```diff
@@ -143,87 +149,86 @@ import '../kb/filters';
         'rec_meta3:meta3_EN_txt',
         'rec_meta4:meta4_EN_txt',
         'rec_date:updatedDate_dt',
-    ]
-	export default {
-        components : { Modal },
-		props:['getDownloadRecords', 'fileName'],
-		data:  () => {
-			return {
-                modal: null,
-				downloadDocs  : [],
-				numFound      : 0,
-				downloadFormat: 'xlsx',
-				loading       : false,
-                isGeneric     : true,
-                schemaFields  : [],
-                schema        : undefined
-			}
-		},
-		created(){
-			// this.articlesApi = new ArticlesApi();
-		},
-		async mounted() {
-            this.modal = new Modal(this.$refs.exportModal);
-        },
-		
-		methods: {
-            async openDialog(){
-                this.modal.show('static');
-                this.loading      = true; 
+    ];
+    const exportModal = ref(null);
```

Comment:
i think for dom `refs` (from line 7) it's better to use `shallowRef` as the reactivity will be only on the reference and not on the dom elelment OR component

https://vuejs.org/api/reactivity-advanced.html#reactivity-api-advanced
```suggestion
    const exportModal = shallowRef(null);
```